features/rest_owner_role.feature

```diff
@@ -0,0 +1,16 @@
+Feature: As a restaurant owner
+  In order to administer my restaurant (menu, dishes etc.)
+  I need a login that gives me restaurant owner rights
+
+  Background:
+    Given the following restaurant owner credentials exist
```

Comment:
This is the same step definition that already exist in the system when you create users. I don't see a need to introduce a new one.